### PR TITLE
Add: Stamina Wheel Display

### PR DESCRIPTION
**Pull request by JPGalovic**

### CODE REVIEW

**Comment by JPGalovic** [comment]
[Player Mechanic - Stamina Shown as Stamina Wheel](https://trello.com/c/C1yKtayw/2-player-mechanic-stamina-shown-as-stamina-wheel)

File: Assets/StaminaSetLevel.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StaminaSetLevel : MonoBehaviour {
```
Comment (xxfast) [inline Assets/StaminaSetLevel.cs:6]: Might need to rename this in the future, but good for now :)

**Review by xxfast — APPROVED** [review]

**Comment by xxfast** [comment]
Good work james :)

**Comment by xxfast** [comment]
I've added the player functionality to a prefab, so we can switch that around in play test

**Pull request merged**